src/Umbraco.Web.UI.Client/src/views/components/content/umb-variant-content.html

```diff
@@ -5,7 +5,7 @@
 
     <div class="umb-split-view__content" ng-show="!vm.editor.loading">
 
-        <ng-form name="contentHeaderForm">
+        <ng-form name="contentHeaderForm" ng-if="vm.editor.content.apps.length > 0">
```

Comment:
Wondering if the ng-if can be changed to ng-show?

Reply:
Nope... for some reason stuff breaks if the `ng-form` is there but just hidden. I did try 😄

Reply:
I completely thought so, but just asking :)